test/array_contraints_test.rb

```diff
@@ -9,4 +9,6 @@
   rejects 123
   rejects [123, 456]
   rejects ['abc', 123]
+  rejects nil
```

Comment:
hmmm rails converts empty arrays to nil, so this will break stuff, how about adding the shady exception to make array also match nil ?

@staugaard opinions ?